new/rfc-knife-bootstrap-client-config.md

```diff
@@ -0,0 +1,213 @@
+---
+RFC: unassigned
+Author: Jay Mundrawala <jdm@chef.io>
+Author: Bryan McLellan <btm@chef.io>
+Status: Draft
+Type: Standards Track
+
+---
+
+# Knife Bootstrap `client.rb` Configuration
+
+Knife bootstrap currently only allows a select few `client.rb` parameters
+to be specified during bootstrap. This RFC proposes a solution to allow users
+to pass bulk configuration data to `knife bootstrap` to be placed in the
+`client.rb` before the first chef-client run.
+
+## Background
+
+The `knife-bootstrap` command supports a small subset of possible parameters
+a user can configure in the `client.rb`. The chef-client cookbook provides a
+template that supports many more, allowing the user to configure additional
+options during a Chef client converge. If something you are trying to do does
+not fit into the template provided, you are also free to specify config files
+in a `client.d` folder that will automatically be loaded.
```

Comment:
You might add, "however, this does not solve the problem" sentence here.

> However, because the chef-client cookbook is run along with the rest of the run list on the first run, your settings do not take affect until the second chef-client run.